README.md

```diff
@@ -2,82 +2,149 @@
+| this project                                                       | The main web application                                                    |
+| [craft-dev](https://github.com/biglotteryfund/craft-dev)           | A headless CMS which powers content pages on the website                    |
+| [grants-service](https://github.com/biglotteryfund/grants-service) | A set of lambda functions which power the past grants search on the website |
 
-- [Express](https://expressjs.com/) running on [Node.js](https://nodejs.org/en/)
-- A MySQL instance running on AWS RDS
-- App running through [Phusion Passenger](https://www.phusionpassenger.com/) via [NGINX](https://www.nginx.com/resources/wiki/)
+You can run the main app without having the other two services set up locally.
 
-## Prerequisites
+## Setup instructions
 
-You'll need the following tools installed:
+### Prerequisites
 
-- [Node.js](https://nodejs.org/en/download/) v8+
-- [Git](https://help.github.com/articles/set-up-git/)
+You'll need the following tools installed in order to run the project locally:
 
-## Setup instructions
+-   Git
+-   Node.js v10+
+-   MySQL v5.7+
+-   [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) configured on your machine
 
-Run these commands in your terminal of choice:
+Once you have the prerequisites installed on your machine, run these commands in your terminal of choice:
 
-### 1. Clone this repository
+### Clone this repository
 
-```
+```shell script
 git clone git@github.com:biglotteryfund/blf-alpha.git
 ```
 
-### 2. Download application secrets
+### Install dependencies:
 
-Next, you'll need to download the application secrets. These are configuration settings that allow the application to external services like a database or email sending service.
+From the root of the project run:
 
-In order to do this, you'll need to:
+```shell script
+npm install
+```
 
-1. [Configure the AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) on your machine
-2. Obtain AWS credentials with permissions to access EC2 Parameter Store
-3. Create a directory called `/etc/blf/` and make sure it is writeable. This is where application secrets will stored.
-4. Run `./bin/get-secrets`. This will download the application secrets to `/etc/blf/parameters.json`.
-5. Create an `.env` in the root of the project and ask another member of the team for the values to go in here. These values are used for local configuration.
+### Download application secrets
 
-### 3. Install dependencies:
+Next, you'll need to download the application secrets. These are configuration settings that allow the application to external services like a database or email sending service.
 
-From the root of the project run:
+Create a directory called `/etc/blf/` and make sure it is writeable. This is where application secrets will stored.
```

Comment:
Maybe add a note that if `/etc/blf/` doesn't exist (should this be `/tnlcf`?) then you'll need to create it first?

Reply:
Is that not implied by "Create a directory"

Reply:
What, am I expected to actually *read* pull requests now?

Reply:
(sorry, missed that – carry on!)